### DISCUSSION TITLE
Conflict with a premium theme and tailwind usage

### DISCUSSION THREAD

**Discussion by elkofy** [primefaces/community#2418] [PrimeVue Templates] [answered]
Hello I have recently purchased a premium theme (apollo) It is still in v3 And I can't get it to work with tailwind somehow all the style goes away when I try to use it. I need to perform a refactorisation going from vuetify can you please provide more information.

**Comment by tugcekucukoglu** [comment, marked as answer]
Hello,

We are planning to complete Apollo for PrimeVue 4 by early September. Until that time you could check https://www.youtube.com/watch?v=Ni_Yecfxv_g
